### PR TITLE
Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by alex1779**
<a name="readme-top"></a>
<div align="center">
  <br/>
  <h1><b>"Portfolio"</b></h1>
  <h2><b>"implement the mobile menu."</b></h2>
  <br/>
</div>

For this milestone of my portfolio website, I will be implementing the mobile menu using newly acquired knowledge of JavaScript.

On mobile devices, the following functions are implemented:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @alex1779 @Nkaleth, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please reduce the height of the toolbar, right now it looks like this:
  image

- [ ] Please fix the issue where the section behind the mobile menu disappears: 
  image
  The menu should be on top of the section with a transparent background, the content shouldn't disappear.


## Optional suggestions

- [ ] I recommend fixing the nav bar so the user can access the menu from any section of the page.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 1]
```diff
@@ -1157,6 +1157,7 @@ ul {
   left: 0;
   mix-blend-mode: multiply;
   position: fixed;
+  backdrop-filter: blur(10px);
 }
 
 .mobile-menu-navbar {
```
Comment (FrederickMih) [inline css/style.css:1163]: - [ ] The hamburger button and the text **My Logo** are hidden by bg images and text respectively. To verify this, let's kindly check the screenshots below: That of Figma and yours on a tabular form. 

| Yours | Figma |
| --- | --- |
| Screenshot from 2022-12-19 21-20-39 |Screenshot from 2022-12-19 21-26-04 |


- Hence, I couldn't click on the hamburger button to check the other functionalities.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand_woman: @alex1779, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights :green_circle: 

- Correct Linters are set up in this repo :heavy_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Alex, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**